packages/smart-contracts/scripts/erc20-swap-to-conversion.ts

```diff
@@ -26,68 +27,19 @@ export async function deploySwapConversion(
   if (!uniswapV2RouterAddresses[hre.network.name] && !args.swapProxyAddress) {
     console.error(`Missing swap router, cannot deploy ${contractName}.`);
   }
+  const requestSwapFees = '5'; // 0.5%
+  const requestFeesCollector = '0x2191eF87E392377ec08E7c08Eb105Ef5448eCED5';
   const deployment = await deployOne<ERC20SwapToConversion>(args, hre, contractName, {
```

Comment:
Shouldn't we remove this file, since it is not used anymore and the deployment was migrated to create2?

Reply:
It is needed when deploying the contract the "classic" way for testing purposes.